src/vm/gdbjit.cpp

```diff
@@ -1029,6 +1030,79 @@ void TypeMember::DumpStaticDebugInfo(char* ptr, int& offset)
     offset += ptrSize + 2;
 }
 
+void FunctionMember::MangleName(char *buf, int &buf_offset, const char *name)
```

Comment:
This simple mangling is problematic. Conside that C# function names can contain even national characters, like characters with various accents. So in case two symbols differ in one character and that character is in both cases a character that your mangling encodes as _, you will generate identical mangled names for two different functions.